### PR TITLE
Update examples/meteor to use API v1beta3

### PR DESCRIPTION
**Pull request by caesarxuchao**
@chees @nikhiljindal 
- I have updated the json files. I have verified that these json files can create service/pod/replicationcontroller correctly. But I don't know how to check if meteor is running correctly. Can some one verify?
- I change the way to get the external IP of the meteor service. Now it uses kubectl instead of the gcloud specific command.

### CODE REVIEW

File: examples/meteor/README.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ Wait until Mongo is started completely and then set up Meteor:
     kubectl create -f meteor-controller.json
     kubectl create -f meteor-service.json
 
-Note that meteor-service.json creates an external load balancer, so your app should be available through the IP of that load balancer once the Meteor pods are started. On gcloud you can find the IP of your load balancer by running:
+Note that meteor-service.json creates an external load balancer, so your app should be available through the IP of that load balancer once the Meteor pods are started. You can find the IP of your load balancer by running:
 
-    gcloud compute forwarding-rules list k8s-meteor-default-meteor | grep k8s-meteor-default-meteor | awk '{print $3}'
+    kubectl get services/meteor -o yaml | awk '/publicIPs/{getline; print $2}'
```
Comment (nikhiljindal) [inline examples/meteor/README.md]: How about: ```kubectl get services/meteor -o template -t "{{.spec.publicIPs}}"```? It seems cleaner.

**Comment by nikhiljindal** [comment]
LGTM modulo one comment.

File: examples/meteor/README.md (hunk 1 above)
Comment (caesarxuchao) [inline examples/meteor/README.md]: Sure. Using template exposes more kubectl features.

**Comment by nikhiljindal** [comment]
LGTM. 
As per https://github.com/Q42/meteor-gke-example/blob/master/README.md, you can verify if it is working fine by verifying that it "prints the public IP address of the machine the app is running on"?

**Comment by chees** [comment]
I verified that following these steps with the v1beta3 api works.

However, at first it took a long time before Mongo was not "pending" anymore and running `gcloud alpha container kubectl describe pod mongo` showed:

    Thu, 07 May 2015 11:24:50 +0200 Thu, 07 May 2015 11:24:50 +0200 1 {kubelet k8s-cluster-2-node-1}            failedMount Unable to mount volumes for pod "mongo_default": Could not attach disk: Timeout after 10s
    Thu, 07 May 2015 11:24:50 +0200 Thu, 07 May 2015 11:24:50 +0200 1 {kubelet k8s-cluster-2-node-1}            failedSync  Error syncing pod, skipping: Could not attach disk: Timeout after 10s

Not sure if that's related to this change. After a while the problem fixed itself and mongo came up anyway, so I guess it's not a serious problem.

**Comment by nikhiljindal** [comment]
Great. Will merge then

**Pull request merged**